### PR TITLE
Add `hide` option support to lighthouse

### PR DESCRIPTION
**Pull request by andymsuk**
Make Lighthouse ignore elements matching those passed in via the `hide` option, just like bbc-a11y.

### CODE REVIEW

File: test/lib/lighthouse.js [hunk 1]
```diff
@@ -222,7 +222,7 @@ describe('lighthouse', () => {
         return lighthouseRunner.run().then(() => {
           sandbox.assert.calledWith(reportBuilder.testSuite().testCase().className, 'www.bbc.co.uk./path/1');
           sandbox.assert.calledWith(reportBuilder.testSuite().testCase().name, '`[role]` values are valid.');
-          sandbox.assert.calledWith(reportBuilder.testSuite().testCase().time, 1200);
+          sandbox.assert.calledWith(reportBuilder.testSuite().testCase().time, 6000 / Object.keys(fakeResults.audits).length);
```
Comment (robtpaton) [inline test/lib/lighthouse.js]: ?

Comment (andymsuk) [reply test/lib/lighthouse.js]: Lighthouse doesn't report a duration for each test, so it's the fake total time from the stubbed lighthouse report (6000) divided by the number of tests.

Comment (DaMouse404) [reply test/lib/lighthouse.js]: You could use variables to clarify what you just said 😸

Comment (andymsuk) [reply test/lib/lighthouse.js]: Touché. Done!

**Pull request merged**